audit-base/pom.xml

```diff
@@ -6,12 +6,12 @@
     <parent>
         <groupId>io.getlime.core</groupId>
         <artifactId>lime-java-core-parent</artifactId>
-        <version>1.5.2</version>
+        <version>1.6.0-SNAPSHOT</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
 
     <artifactId>audit-base</artifactId>
-    <version>1.5.2</version>
+    <version>1.6.0-SNAPSHOT</version>
```

Comment:
I would rather omit the version completely. It is taken from parent.

Reply:
Ok.